src/ElasticSearch.php

```diff
@@ -28,6 +28,14 @@ public function create($collection)
 
     public function drop($collection)
     {
+        $params = ['index' => $this->index];
+        try {
+            $this->conn->indices()->delete($params);
+        } catch (\Exception $e) {
+            // This ignore the error
+            return false;
+        }
+        return true;
```

Comment:
@hkulekci  Test on Travis couldn't pass due to expression about hhvm in the php versions section in the travis.yml. Since we drop the hhvm support, could you please remove hhvm line from travis.yml and send PR again?